README.md

```diff
@@ -71,13 +69,13 @@ So lets do it the old fashion way:
 
 ### Helpful hints
 
-If you are using any protected nuget feeds then run the following command to add your credentials
+If you are using any protected nuget feeds, run the following command to add your credentials:
 
-    condo -SecureFeed
+    ```condo -SecureFeed```
```

Comment:
This is Windows specific; the *nix way of doing this is different (and currently more annoying).